pallets/api/src/fungibles/tests.rs

```diff
@@ -504,11 +508,14 @@ fn token_metadata_works() {
 			ReadResult::TokenDecimals(Default::default())
```

Comment:
Would be nicer to use None in the above examples rather than default to be more explicit: `assert_eq!(Fungibles::read(TokenName(TOKEN)), ReadResult::TokenName(None));`, as in the integration tests.

Comment only, not a requirement.